packages/wallets/src/features/accounts/modules/DocumentService/components/IDVService/IDVService.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useEffect, useState } from 'react';
+import { Formik } from 'formik';
+import { Loader } from '@deriv-com/ui';
+import { FormDropdown, FormField, ModalStepWrapper, WalletText } from '../../../../../../components';
+import { Footer } from '../../../components';
+import {
+    TVerifyPersonalDetailsValues,
+    useVerifyPersonalDetails,
+    VerifyPersonalDetails,
+} from '../VerifyPersonalDetails';
+import { IDVServiceErrorMessage } from './components';
+import { useIDVService } from './hooks';
+import { TIDVServiceValues } from './types';
+import { documentTypeValidator, getDocumentNumberValidator } from './utils';
+import './IDVService.scss';
+
+type TIDVServiceProps = {
+    onCompletion?: VoidFunction;
+    onDocumentNotAvailable?: VoidFunction;
+};
+
+const IDVService: React.FC<React.PropsWithChildren<TIDVServiceProps>> = ({ onCompletion, onDocumentNotAvailable }) => {
+    const {
+        availableDocumentOptions,
+        displayedDocumentsList,
+        documentExamples,
+        error: errorIDVDetails,
+        initialValues: initialIDVValues,
+        isLoading: isIDVDataLoading,
+        isSubmitted: isIDVSubmitted,
+        isSubmitting: isIDVSubmitting,
+        previousSubmissionErrorStatus,
+        submit: submitIDVDetails,
+    } = useIDVService();
+    const {
+        error: errorPersonalDetails,
+        initialValues: initialPersonalDetailsValues,
+        isLoading: isPersonalDetailsDataLoading,
+        isSubmitted: isPersonalDetailsSubmitted,
+        isSubmitting: isPersonalDetailsSubmitting,
+        submit: submitPersonalDetails,
+    } = useVerifyPersonalDetails();
+    const [clientHasDocuments, setClientHasDocuments] = useState<boolean>(true);
+
+    const isDataLoading = isIDVDataLoading || isPersonalDetailsDataLoading;
+    const isSubmitting = isIDVSubmitting && isPersonalDetailsSubmitting;
+    const isSubmitted = isIDVSubmitted && isPersonalDetailsSubmitted;
+
+    const errorMessage = previousSubmissionErrorStatus ?? errorIDVDetails?.code ?? errorPersonalDetails?.code;
+
+    const submit = (values: TIDVServiceValues & TVerifyPersonalDetailsValues) => {
+        if (clientHasDocuments) {
+            submitPersonalDetails(values);
+            submitIDVDetails(values);
```

Comment:
Hi @wojciech-deriv, 
Since there are two subforms (IDV and Personal details) which share the same <Formik /> wrapper, the values contains the data for both. But these functions are aware to pull their respective values and make a call.